### DISCUSSION TITLE
The notification was sent to me incorrectly

### DISCUSSION THREAD

**Discussion by DrAugus** [community/community#42817] [Discussions] [answered]
### Select Topic Area

Bug

### Body

![1672292672357](https://user-images.githubusercontent.com/34206897/209908469-b11c27c8-7214-4cb5-8574-5ca2802583e0.png)

I was not mentioned at all in this discussion.

https://github.com/Clinical-Genomics/scout/discussions/3743#discussioncomment-4529048

**Comment by Akash1134** [comment, marked as answer]
Hi @DrAugus, GitHub is responding to concerns expressed by community members regarding recent spam posts & comments. Thanks for sharing and we appreciate your patience.
